src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsRepeatableIncontentBoxadMobileWiki.java

```diff
@@ -60,15 +60,15 @@ public void adDisplayedBeforeSections() {
                          "IncontentBoxad is not displayed before section"
     );
     ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(2))));
-    Assertion.assertFalse(isIncontenAdDisplayed(3, ads),
+    Assertion.assertFalse(isIncontenAdDisplayed(0, ads),
```

Comment:
Why? It isn't the same scenario

Reply:
This slot has no boxad. I didn't wanted to change entire logic to adjust to it